server/bundler/loader.js

```diff
@@ -86,7 +86,7 @@ function splitTemplate( path, module, chunkName ) {
 		'		}',
 		'		context.store.dispatch( { type: "SET_SECTION", isLoading: false } );',
 		'		if ( ! _loadedSections[ ' + JSON.stringify( module ) + ' ] ) {',
-		'			require( ' + JSON.stringify( module ) + ' )( controller.clientRouter );',
+		'			require( ' + JSON.stringify( module ) + ' )( controller.clientRouter, controller.clientRenderer );',
```

Comment:
Not sure I understand the question... We're passing these functions here as arguments for indiviual sections' controllers to use, but they can entirely ignore them (all sections except for ours do so far). [`controller`](https://github.com/Automattic/wp-calypso/blob/update/decouple-client-router-and-renderer/server/bundler/loader.js#L16) here, OTOH, is `client/controller.js`, which exports `clientRouter` and `clientRenderer`.